forge-gui/res/cardsfolder/upcoming/lux_artillery.txt

```diff
@@ -0,0 +1,11 @@
+Name:Lux Artillery
+ManaCost:4
+Types:Artifact
+T:Mode$ SpellCast | ValidCard$ Artifact.Creature | ValidActivatingPlayer$ You | Execute$ TrigAnimate | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast an artifact creature spell, it gains sunburst. (It enters the battlefield with a +1/+1 counter on it for each color of mana spent to cast it.)
+SVar:TrigAnimate:DB$ Animate | Keywords$ Sunburst | Defined$ TriggeredCard | Duration$ Permanent
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | ConditionCheckSVar$ X | ConditionCompare$ GE30 | Execute$ TrigDealDamage | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your end step, if there are thirty or more counters among artifacts and creatures you control, CARDNAME deals 10 damage to each opponent.
+SVar:TrigDealDamage:DB$ DealDamage | Defined$ Opponent | NumDmg$ 10
+SVar:X:Count$Valid Artifact.YouCtrl,Creature.YouCtrl$CardCounters
```

Comment:
you need to test that
I think it could either crash or justnot work

Reply:
I don't think it worked if you didn't use "ALL"

Reply:
Yes, I have tested it with All and it seems to work now

Reply:
Doesn't seem correct to me :p

![grafik](https://user-images.githubusercontent.com/8506892/214547091-575cfb59-ab28-4ac0-882e-440cd4899262.png)

Reply:
Oh I see tested it with thirty counters on board

Reply:
Turns out All needs to be ALL